scripts/dev/misc_checks.py

```diff
@@ -226,6 +231,10 @@ def check_spelling(args: argparse.Namespace) -> Optional[bool]:
             re.compile(r'IOError'),
             "use OSError",
         ),
+        (
+            re.compile(r'qtbot\.(?!{})[a-z]+[A-Z].*'.format(excluding)),
```

Comment:
You can use [f-strings](https://realpython.com/python-f-strings/) instead:

```suggestion
            re.compile(fr'qtbot\.(?!{excluding})[a-z]+[A-Z].*'),
```